src/wallet/rpcwallet.cpp

```diff
@@ -1849,9 +1849,77 @@ UniValue backupwallet(const UniValue& params, bool fHelp)
 
     bool bWalletBackupResults = BackupWallet(*pwalletMain, GetBackupFilename("wallet.dat"));
     bool bConfigBackupResults = BackupConfigFile(GetBackupFilename("gridcoinresearch.conf"));
+
+    std::vector<std::string> backup_file_type;
+
+    backup_file_type.push_back("wallet.dat");
+    backup_file_type.push_back("gridcoinresearch.conf");
+
+    std::vector<std::string> files_removed;
+    UniValue u_files_removed(UniValue::VARR);
+
+    bool bManageBackupResults = ManageBackups(GetBackupPath(), backup_file_type, 0, 0, files_removed);
+
+    for (const auto& iter : files_removed)
+    {
+        u_files_removed.push_back(iter);
+    }
+
     UniValue ret(UniValue::VOBJ);
     ret.pushKV("Backup wallet success", bWalletBackupResults);
     ret.pushKV("Backup config success", bConfigBackupResults);
+    ret.pushKV("Manage backup file retention success", bManageBackupResults);
+    ret.pushKV("number_of_files_removed", (int64_t) files_removed.size());
```

Comment:
Probably want to keep the keys here consistent and use the same styling. It looks like the other keys don't use underlines
```suggestion
    ret.pushKV("Number of files removed", (int64_t) files_removed.size());
```

Reply:
Because the other keys are the original keys from the function before the addition. The question is whether people have existing scripts that will break if I change them.

Reply:
See my comments below on this.